pom.xml

```diff
@@ -355,7 +355,8 @@ limitations under the License.
           <version>3.2.2</version>
           <configuration>
             <configLocation>config/maven_checks.xml</configLocation>
-            <headerLocation>https://raw.github.com/codehaus-plexus/plexus-pom/master/src/main/resources/config/plexus-header.txt</headerLocation>
+            <!-- version from release 10 referenced by branch -->
+            <headerLocation>https://github.com/codehaus-plexus/plexus-pom/blob/plexus-10/src/main/resources/config/plexus-header.txt</headerLocation>
```

Comment:
maybe spotless will be better here

Reply:
Agree that we should move to Spotless soon, but if someone will just remove plexus-header.txt file from master branch something very bad may happen ;-)